string name                 # node name
int8 state                  # 0: Running, 1: Warning, 2: Crashed
string publication_rate     # publication rate of node